packages/fast-components-react-msft/src/carousel/carousel.tsx

```diff
@@ -292,7 +319,7 @@ class Carousel extends Foundation<
      * Generates previous flipper if more than one slide
      */
     private generatePreviousFlipper(): any {
-        if (!this.isMultipleSlides) {
+        if (!this.isMultipleSlides || (this.isFirstSlide && !this.props.loop)) {
```

Comment:
I think you should do:
`(!this.isMultipleSlides || (this.isMultipleSlides && this.isFirstSlide && !this.props.loop))`

Because you probably only want to render this if its the first slide, not looping AND there are multiple slides.

Reply:
Actually, I've just realized that's redundant, nevermind.